### PR TITLE
Add Microsoft HoC blocks back to /minecraft

### PR DESCRIPTION
**Pull request by joshlory**
Reverts code-dot-org/code-dot-org#12421.

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/minecraft.haml [hunk 1]
```diff
@@ -51,6 +51,27 @@ style_min: true
...
 
+.color-block.mc-store
+  %h2.white-text= I18n.t(:minecraft_store_camps)
+  = I18n.t(:minecraft_signup_store_camps).gsub('%{signup_store_camp}', 'https://www.microsoft.com/youthsparkprograms')
+
+.color-block.mc-host-event
+  %h2.white-text= I18n.t(:minecraft_host_event)
+  %a{href: '/hourofcode/mc'}= I18n.t(:minecraft_teach_resources)
+  %br
+  %a{href: 'https://docs.com/hourofcode2016'}= I18n.t(:minecraft_leader_guide)
```
Comment (joshlory) [inline pegasus/sites.v3/code.org/public/minecraft.haml]: @tanyaparker Do these resources need to be updated for 2017?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/minecraft.haml]: Yes, that looks old. Have we shared back these boxes with Microsoft? @ryansloan I think we need to update these resources?

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/minecraft.haml]: All the other links seem to be ok to me.

**Comment by tanyaparker** [comment]
**BOX 1: Cyan #0094ca**

* Title: Free Minecraft Hour of Code Camps at the Microsoft Store
* Description: [Sign up](https://www.microsoft.com/youthsparkprograms) for the free Minecraft Hour of Code workshops at Microsoft Stores in Australia, Canada, Puerto Rico or the US.  Learn how coding is for everyone and combines creativity, problem solving and loads of fun.
 
**BOX 2: Green #b9bf15**

* Title: Host a Minecraft Hour of Code Event
* Description: Join us on November 16th, 2017 for a special webinar for those who want to facilitate Hour of Code with Minecraft. Get a walkthrough of the new tutorial by the Minecraft team and learn about the resources and best practices to help you confidently facilitate an exciting Hour of Code with Minecraft. [Sign up today](https://aka.ms/mcheroworkshop)!
* Description (2nd paragraph): [Download](https://code.org/minecraft/facilitator) facilitator quick start guide and PowerPoint presentations for younger and older students now.

 
**BOX 3: Purple #7665a0**

* Title: Computer Science for Everyone: Beyond an Hour of Code at the Microsoft Store
* Description: No computer science experience required! Help ensure all youth can learn the valuable skills gained through computer science education. Join us at the Microsoft Store to engage in a free series of three, 1-hour workshops where you will experience coding in a fun and stress-free environment, become familiar with key computer science concepts, and get ready to lead an Hour of Code in your classroom or community. You got this! [Sign up today](https://www.microsoft.com/en-us/store/locations/events-for-educators).


**BOX 4: Teal #00adbc**

* Title: Help all youth become future ready
* Description: Find out how other schools and nonprofits are bringing digital skills, from digital literacy to computer science education, to their communities. Consider different models, get tips and tricks, and [access the resources](https://www.microsoft.com/digitalskills) they found most valuable.

**Comment by joshlory** [comment]
Boxes are a little unbalanced due to length of strings.  Should (2) and (3) be full-width, sandwiching (1) and (4) between them?

screen shot 2017-11-10 at 2 10 46 pm

**Pull request merged**